src/commands/suggestions/AddList.ts

```diff
@@ -0,0 +1,67 @@
+import {Command, CommandoClient, CommandoMessage} from "discord.js-commando";
+import {Message} from "discord.js";
+import {SuggestionsManager} from "../../utility/SuggestionsManager";
+
+/**
+ * Enum for potential results
+ * @constant
+ */
+const resultEnum = {LISTEXISTS: 6, SUGGESTIONDOESNOTEXIST: 5, LISTADDED: 4, DERP: 3, SUGGESTIONADDED: 2, LISTDOESNOTEXIST: 1, SUGGESTIONEXISTS: 0};
```

Comment:
Made a bunch of comments on enums in AddSuggestion.ts, those also apply here; suggest moving this to SuggestionManager as an enum type rather than redefining here.